Util.java

```diff
@@ -13,12 +13,24 @@
 	 *
 	 * @param var
 	 *        The floating point number you want to compare
+	 * @param epsilon
+	 *		  The maximum value of var such that it can be considered zero.
 	 * @return
-	 * 		Whether or not it is within Double.MIN_VALUE of zero
+	 * 		Whether or not var is within epsilon of zero
 	 */
 	public static boolean isZero(double var, double epsilon) {
 		return Math.abs(var) < epsilon;
 	}
+
+	/**
+	 * A function for determining if floating point numbers are effectively zero.
+	 * Floating point arithmetic tends to introduce very small errors.
```

Comment:
The original documentation for this function is a little meh. We could improve both instances of this summary to something like "Returns true if `{@code var}` is effectively zero. This is useful for floating point numbers, which tend to incur small errors through arithmetic operations." This is just a suggestion, so go with however you would prefer to phrase it.